providers/aix/process_aix_ppc64.go

```diff
@@ -218,12 +217,12 @@ func (p *process) Environment() (map[string]string, error) {
 			break
 		}
 		if err != nil {
-			return nil, errors.Wrap(err, "error while calling getevars")
+			return nil, fmt.Errorf("error while calling getevars: %w", err)
 		}
 
 		pair := bytes.SplitN(chop(line), delim, 2)
 		if len(pair) != 2 {
-			return nil, errors.Wrap(err, "error reading process environment")
+			return nil, fmt.Errorf("error reading process environment: %w", err)
```

Comment:
This looks like it was a bug because it would have returned a `nil` error previously. The `err` has already been handled. So I think the message should be something like `errors.New("error reading process environment: unexpected format")`.

Reply:
Good catch! Updated :+1: